bears/python/VultureBear.py

```diff
@@ -3,6 +3,19 @@
 from dependency_management.requirements.PipRequirement import PipRequirement
 from vulture import Vulture
 
+CONFIDENCE_MAP = {
+    'attribute': 50,
+    'class': 50,
+    'function': 50,
+    'import': 100,
+    'property': 50,
+    'variable': 50,
+}
+
+
+def calc_confidence(typ):
```

Comment:
No need for this function. I think we can directly use `CONFIDENCE_MAP[item.typ]` below. We don't want to use get() since it swallows missing confidence values silently.